### PR TITLE
Add support for write-file on x86

### PR DESCRIPTION
**Pull request by disaykin**

### CODE REVIEW

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: src/libinjector/win_injector.c [hunk 1]
```diff
@@ -705,16 +721,16 @@ static event_response_t wait_for_target_process_cb(drakvuf_t drakvuf, drakvuf_tr
     if (info->proc_data.pid != injector->target_pid)
         return 0;
 
+    if (injector->target_tid && injector->target_tid != (uint32_t)info->proc_data.tid)
+        return 0;
+
```
Comment (BonusPlay) [inline src/libinjector/win_injector.c:726]: Why is this moved?

File: src/libinjector/win_injector.c [hunk 2]
```diff
@@ -1350,7 +1366,6 @@ static event_response_t injector_int3_cb(drakvuf_t drakvuf, drakvuf_trap_info_t*
                 break;
             default:
                 // TODO Implement
-                success = false;
```
Comment (BonusPlay) [inline src/libinjector/win_injector.c:1353]: ?

File: src/libinjector/win_injector.c [hunk 3]
```diff
@@ -672,7 +681,14 @@ static event_response_t mem_callback(drakvuf_t drakvuf, drakvuf_trap_info_t* inf
...
     regs.x86.rip = injector->exec_func;
-    injector->status = STATUS_CREATE_OK;
+    if (INJECT_METHOD_WRITE_FILE == injector->method)
+    {
+        injector->status = STATUS_ALLOC_OK;
+    }
+    else
+    {
+        injector->status = STATUS_CREATE_OK;
+    }
```
Comment (BonusPlay) [inline src/libinjector/win_injector.c:691]: please annotate that it's `CREATE_FILE` or `SHELLEXEC`

File: src/libinjector/win_injector.c (hunk 1 above)
Comment (disaykin) [reply src/libinjector/win_injector.c:732]: I move fast check before slow one.

File: src/libinjector/win_injector.c (hunk 2 above)
Comment (disaykin) [reply src/libinjector/win_injector.c:1353]: See the statement before switch.

`bool success = false;`

File: src/libinjector/win_injector.c (hunk 3 above)
Comment (disaykin) [reply src/libinjector/win_injector.c]: Done.
